django/thunderstore/cache/cache.py

```diff
@@ -21,6 +21,10 @@ class CacheBustCondition(ChoiceEnum):
     dynamic_html_updated = "dynamic_html_updated"
 
 
+# Move this here to avoid circular import of CacheBustCondition
```

Comment:
A better option would be to move `CacheBustCondition` to it's own standalone file